### PR TITLE
this address issue #170

### PR DESCRIPTION
**Pull request by ChrisCPO**
Fixes issue(s) #170.

Changes proposed in this pull request:
/cc relevant people
- allows a local developer to log-in;
- bypasses oauth
- see .sample.env
- set developer email
- set skip_team_check to 'true'

### CODE REVIEW

File: app/controllers/auth_controller.rb [hunk 1]
```diff
@@ -2,16 +2,36 @@ class AuthController < ApplicationController
...
-      user = User.find_or_create_by(email: auth_email)
-      sign_in(user)
-      flash[:success] = "You successfully signed in"
-      redirect_to root_path
+    if ok_to_bypass_authentication?
+      create_user(developer_email)
+    else
+      if team_member?
+        create_user(auth_email)
+      end
```
Comment (afeld) [inline app/controllers/auth_controller.rb]: Not a blocker for this pull request, but there should probably be an `else` here.

**Comment by afeld** [comment]
This looks like it will be very helpful! Some TODOs:

* [ ] Try without the GitHub credentials set (i.e. avoiding those parts of the setup)
* [ ] Mention how to use this mode in CONTRIBUTING.md
* [ ] Consolidate the two environment variables down to one (e.g. `SIGN_IN_AS`)

Also, as a convention, it's useful to the project team to make the commit and pull request title something more descriptive, like "added a way to bypass authentication." This makes it easier to understand what the actual change was when looking at the commit history. Thanks again!

**Comment by ben-biddington** [comment]
@afeld would you like tests for this stuff?

**Comment by ChrisCPO** [comment]
@ben-biddington @afeld  Hey will add some tests and edit changes. Virtual box overwrote my desktop packages so had to deal with that on Sunday. Should be able to do this evening.

**Comment by jessieay** [comment]
Hi there @ChrisCPO - do you have time to update this?

**Comment by jessieay** [comment]
Voting to close this without merge since it's over a year stale. Can be re-opened if someone wants to take the time to update it.

**Comment by jmhooper** [comment]
@jessieay: Agreed

**Pull request closed without merging**